docs/en-US/troubleshooting/version.md

```diff
@@ -0,0 +1,35 @@
+---
+category: 5. Troubleshooting
+order: 3
+title: Versioning system used by VVV
+description: How VVV project manage the various version and upgrades for the users
+permalink: /docs/en-US/troubleshooting/versioning-system/
+---
+
+VVV projects is using the [semver](http://semver.org/) standard on defined and managing the various releases.
+
+
+## What does it means
+ 
+VVV is a suite of different tools that let you to manage different websites automatically with the various needs.  
+Keep everything updated and aligned is not an easy task so to simplify the environment management the semver standard helps you and the contributors to the project.  
+WordPress as example doesn't use the semver for their releases so every release can have different changes or backward compatibilities.
+For any other kind of issues don't forget to check at [Troubleshooting page](https://varyingvagrantvagrants.org/docs/en-US/troubleshooting/).
+
+### Major versions
+
+With major version we are talking of 1.x.x or 2.x.x that in the VVV case require a `vagrant destroy` to download new virtual image and reinstall everything.  
+Commands to run to update to this version:
+
+    git pull # update to latest git version
```

Comment:
I don't think we need to document how to update, we have a page for that, and the instructions shouldn't matter wether it's a major or minor release.

Reply:
I left that part because the ticket mention it and in that way is more clear what to do in both of the cases instead have a way that include vagrant destroy for everything (like the actual doc has).

Reply:
I checked the desotry is mentioned only https://varyingvagrantvagrants.org/docs/en-US/troubleshooting/ about strating from fresh and not for updates.